specification/web/resource-manager/Microsoft.DomainRegistration/stable/2024-11-01/Domains.json

```diff
@@ -0,0 +1,1616 @@
+          "enum": [
+            "Regular",
+            "SoftDeleted"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "DomainType",
+            "modelAsString": false
+          }
+        }
+      }
+    },
+    "DomainCollection": {
+      "description": "Collection of domains.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Domain"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "DomainControlCenterSsoRequest": {
+      "description": "Single sign-on request information for domain management.",
+      "type": "object",
+      "properties": {
+        "url": {
+          "description": "URL where the single sign-on request is to be made.",
+          "type": "string",
+          "readOnly": true
+        },
+        "postParameterKey": {
+          "description": "Post parameter key.",
+          "type": "string",
+          "readOnly": true
+        },
+        "postParameterValue": {
+          "description": "Post parameter value. Client should use 'application/x-www-form-urlencoded' encoding for this value.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "DomainOwnershipIdentifier": {
+      "description": "Domain ownership Identifier.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../Microsoft.Web/stable/2024-11-01/CommonDefinitions.json#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "DomainOwnershipIdentifier resource specific properties",
+          "type": "object",
+          "properties": {
+            "ownershipId": {
+              "description": "Ownership Id.",
+              "type": "string"
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "DomainOwnershipIdentifierCollection": {
+      "description": "Collection of domain ownership identifiers.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DomainOwnershipIdentifier"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "DomainPatchResource": {
```

Comment:
>DomainPatchResource [](http://example.com/codeflow?start=5&length=19)

This payload includes many read-only properties, which is not allowed in a PATCH payload.